EditFAQs.ascx.cs

```diff
@@ -99,10 +99,14 @@ private void PopulateCategoriesDropDown()
         /// <param name="e">The <see cref="System.EventArgs" /> instance containing the event data.</param>
         protected void Page_Load(System.Object sender, System.EventArgs e)
         {
+            if (this.Page.ClientScript.IsClientScriptBlockRegistered("pikaday.min.js") == false)
+            {
+                this.Page.ClientScript.RegisterClientScriptBlock(this.GetType(), "Moment.js", "<script language=javascript src=\"https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js\"></script>");
+                this.Page.ClientScript.RegisterClientScriptBlock(this.GetType(), "AjaxFaq.js", "<script language=javascript src=\"https://cdn.jsdelivr.net/npm/pikaday/pikaday.min.js\"></script>");
+            }
```

Comment:
This relies exclusivelly on cdns which could not work in some intranets with external internet access disabled

Reply:
`moment` is deprecated.  It is a pretty easy swap for `dayjs`.  I am happy to help with that if desired.  I can also help with converting this away from CDN.  Since this is not a frontend project, I guess we would use local versions of the libraries (like we do in DNN)?